modules/ROOT/pages/creating-and-managing-a-cluster-manually.adoc

```diff
@@ -58,16 +50,14 @@ Manual management of a cluster is only possible for clusters that have been manu
 To manually change the configuration of a cluster node, follow these steps:
 
 . Stop the node's Mule server.
-
 . Edit the node's `mule-cluster.properties` as desired, then save the file.
-
 . Restart the node's Mule server.
 
 [CAUTION]
 ====
 *Ensure consistency across nodes*
 
-Ensure that the options you apply in the configuration file are valid for all cluster nodes. Failure to do so can cause you to  break the cluster configuration and inadvertently disable the cluster.
+Ensure that the options you apply in the configuration file are valid for all cluster nodes. Failure to do so can cause you to break the cluster configuration and inadvertently disable the cluster.
 ====
```

Comment:
Hmm...I don't know about using a notice format AND beginning it with a bold element.

Also, that bold element is a sentence and should end with a period, unless you are treating it as a heading, in which case it should be in all init. caps.

And is consistency the right word, or are we really talking about validity?

How about...:
```suggestion
[CAUTION]
====
Failure to ensure that the options you apply in the configuration file are valid for all cluster nodes can break cluster configuration and disable the cluster.
```